doc/nrf/libraries/nrf_rpc/nrf_rpc_ipc.rst

```diff
@@ -8,8 +8,7 @@ nRF RPC IPC Service transport
    :depth: 2
 
 The nRF RPC IPC Service transport allows you to use the :ref:`nrf_rpc` library to execute procedures on remote processors.
-
-This feature is used in the :ref:`ble_rpc` library and also in the :ref:`nrf_rpc_entropy_nrf53` sample.
```

Comment:
Why did you remove mentioning of `ble_rpc`?

Reply:
Because `ble_rpc` can also use UART, depending on the configuration. Should I rephrase to something like "This feature is used in the :ref:`nrf_rpc_entropy_nrf53` sample and can also be enabled for `ble_rpc`"?

Reply:
Why I would like to leave here any library mentioning (probably it is worth to add nfc and open thread too)? This feature is much wider purposed rather than just sample. Reading this, customer should be aware of abilities and purposes.

Reply:
I personally find it strange to list all components that use a given library. A library is by definition something that is not limited to a specific set of use cases so this always creates a risk that some component is missed and confuses me as a reader: why this doc only mentions A and not B if both A and B use the library in the same way.
But if you disagree, I'll revert the change.

Reply:
We could also split the difference, and have:
"This feature is used in the :ref:`nrf_rpc_entropy_nrf53` sample, and several |NCS| libraries, such as :ref:`ble_rpc`."

By framing the mentioned library (or libraries, if you wish) as examples, it makes it clear the list isn't exhaustive.